docs/UserGuide.md

```diff
@@ -139,14 +139,13 @@ Examples:
 
 ### Filtering persons: `find`
 
-* Filter persons based on specific criteria within their records.
+Filter persons based on specific criteria within their records.
 
 Format: `find PREFIX/KEYWORD`
 
 * This command searches for persons using a specific aspect of their details, as specified by the prefix.
 * The search will return any result that contains the keyword as a substring under the indicated prefix. e.g. `find e/hans` will find any person that contains `hans` in their email.
 * The search is case-insensitive. e.g. `hans` will match `Hans`.
-* prefixes that are supported includes : n/, p/, e/, a/, m/ and t/.
```

Comment:
Good job! Now users are aware that they are able to find contacts through so many fields.